pkg/cache/cache.go

```diff
@@ -27,13 +26,16 @@ var (
 
 type localCache struct{}
 
-func (localCache) getDecision(clientIP string) (bool, error) {
+func (localCache) getDecision(clientIP string) (bool, bool, error) {
```

Comment:
what do you think to expose these 3 variables (Capitalize the first letter) and this function return the bannedString like `func (localCache) getDecision(clientIP string) (string, error) {`
CacheBannedValue   = "t"
CacheNoBannedValue = "f"
CacheCaptchaValue  = "c"

Reply:
that make sense